pkg/lib/config/testdata/config_tests.yaml

```diff
@@ -605,6 +607,7 @@ config:
     public_origin: http://test
   ui:
     default_client_uri: nonsense
+    branding_page_uri: nonsense
```

Comment:
@mickmak-busicat The commit for fixing the name is missing!

Reply:
@louischan-oursky added back :adore2x: